deep4cast/models/__init__.py

```diff
@@ -0,0 +1,3 @@
+# -*- coding: utf-8 -*-
+from .multivariate_rnns import *
```

Comment:
Having `multivariate` prefix before rnn is not necessary. 

And for module/function names, we typically don't add `s` (i.e. making them plural).

Reply:
got it.